.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "esbonio.sphinx.confDir": ""
```

Comment:
Are these `.vscode` files necessary?

Reply:
As discussed on the Discord, please omit anything that automatically suggests extensions, but document how to set them up. I suggest putting this documentation in a separate .rst file and just linking to it in the README.